src/php/PhelAbstractConfig.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel;
+
+use Gacela\Framework\AbstractConfig;
+use Gacela\Framework\Config;
+use function json_decode;
+
+/**
+ * This is a layer on top of Gacela config, to get the values from the composer.json.
+ */
+abstract class PhelAbstractConfig extends AbstractConfig
```

Comment:
I should be named `AbstractPhelConfig`. See: https://github.com/phel-lang/phel-lang/blob/master/adrs/php/0001_naming_conventions.md

Reply:
Totally 👍